ODLA/platforms/odla_popart/odla_popart.h

```diff
@@ -64,8 +64,7 @@ struct _odla_value {
 
 struct _odla_computation {
   std::unique_ptr<popart::Builder> builder;
-  std::map<popart::TensorId, std::unique_ptr<popart::IArray>> inputs;
-  std::map<popart::TensorId, std::unique_ptr<popart::IArray>> outputs;
+  std::unique_ptr<popart::InferenceSession> session;
```

Comment:
can we have `odla_context` in `odla_computation` and `session` is saved in `_odla_context`?
like 
```
struct _odla_computation {
   odla_context ctx;
   ...
};
```


Then, in `odla_DestroyContext(ctx)`, we can do  
```
// reset ctx->session
ctx->comp->ctx = nullptr; 
```

In `odla_DestroyComputation(comp)`, we do 
```
if (comp->ctx) { odla_DestroyContext(comp->ctx); } 
delete (comp);
```

Reply:
I didn't change this based on our discussion in the morning. If I understand our conclusion correctly. I keep `session` in computation and do the destruction as before...Please correct me if I'm wrong.